force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +46,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+        Boolean vote = false;
+        if (age >= 18 ) {
+            vote = true;
+        }
+        return vote; 
```

Comment:
Good. You can solve these problems with one line and no variable thought. It can make your life a little easier. Think about what expressions with. < or > will evaluate too.